Added Grafana questions and answers

AbyelT opened the pull request:
Adding answers to questions in https://github.com/bregman-arie/devops-exercises/issues/4. Also added new questions and answers

AbyelT committed:
Add grafana Q&A

AbyelT committed:
Refactor answers, add reference

AbyelT committed:
New Q&A

AbyelT commented:
Hello @bregman-arie, i added some answers to existing questions from https://github.com/bregman-arie/devops-exercises/issues/4 and created some new questions about using and organizing an Grafana instance. I'm not fully done yet (some typo checking, references) but would love to hear feedback if you have any. Thanks in advance :)

bregman-arie commented on exercises/grafana/README.md:
```diff
@@ -5,3 +5,154 @@
+</b></details>
+
+<details>
+<summary>How can we install plugins</summary><br><b>
```
How can we install plugins for Grafana?*

bregman-arie commented on exercises/grafana/README.md:
```diff
@@ -5,3 +5,154 @@
+</b></details>
+
+<details>
+<summary>How is the version upgrade done</summary><br><b>
```
Same as "on which OS Grafana is supported?"

bregman-arie reviewed (changes requested):
Thanks for the effort. Left some comments.

AbyelT committed:
Remove, Modify questions & answers

- Updated some answers with additional text
- Removed questions "which OS can we install Grafana" and "version upgrade"
- Split Question "difference between Default configuration to Custom configuration" into two separate
- Fixed grammar on some questions

AbyelT commented:
Thanks for the feedback! I agree that some questions are redundant and the user can check the docs instead. I think the Q&A for Grafana is ready to merge, feel free to let me know if there is something to be addressed. Thanks in advance!

bregman-arie commented:
Thanks for the effort
